src/index.html

```diff
@@ -11,6 +11,32 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="form-search-big"
+      action="POST"
```

Comment:
https://www.w3schools.com/tags/att_form_action. "POST" that's a method, not an action

Reply:
changed to method